schema.yaml

```diff
@@ -89,28 +89,28 @@ schema;game:
 schema;genres:
   type: str
   enum: [
+    'Action',
+    'Adventure',
+    'Arcade',
+    'Artillery',
+    'Educational',
+    'Fighting',
     'FPS',
     'MMORPG',
     'MOBA',
+    'Platform',
+    'Puzzle',
+    'Racing',
     'RPG',
     'RTS',
+    'Sports',
     'TBS',
     'TPS',
-    'action',
-    'adventure',
-    'arcade',
-    'artillery',
-    'educational',
-    'fighting',
-    'platform',
-    'puzzle',
-    'racing',
-    'rhythm',
-    'roguelike',
-    'shmup',
-    'simulation',
-    'sports',
-    'horror',
+    'Rhythm',
+    'Roguelike',
+    'Shmup',
+    'Simulation',
+    'Strategy',
   ]
 
 schema;subgenres:
```

Comment:
@tukkek let's run with this. If somebody wants to put the extra effort in here, then I'm not going to stand in their way. What would a Law & Order game remake fall into?